Compiler/script2/cs_parser.cpp

```diff
@@ -4199,7 +4199,7 @@ void AGS::Parser::ParseVardecl_Import(Symbol var_name)
     }
 
     _sym[var_name].VariableD->TypeQualifiers[TQ::kImport] = true;
-    int const import_offset = _scrip.FindOrAddImport(_sym.GetName(var_name));
+    int const import_offset = _scrip.FindOrAddImport(_sym.GetName(var_name).c_str());
```

Comment:
This change looks unnecessary, FindOrAddImport accepts `const std::string&", why this is done?

Reply:
this is probably a mistake, will fix later, for now I just want to get it building so I can switch back to the Windows PC and polish the commits there...

Reply:
fixed